etc/enhancements/viirs.yaml

```
@@ -1,4 +1,18 @@
 enhancements:
+  adaptive_dnb:
+    sensor: viirs
+    standard_name: equalized_radiance
+    operations:
+      - name: linear_stretch
+        method: !!python/name:satpy.enhancements.stretch
+        kwargs: { stretch: 'crude', min_stretch: 0.0, max_stretch: 1.0 }
+  histogram_dnb:
+    sensor: viirs
+    standard_name: equalized_radiance
+    operations:
+      - name: linear_stretch
+        method: !!python/name:satpy.enhancements.stretch
+        kwargs: { stretch: 'crude', min_stretch: 0.0, max_stretch: 1.0 }
```

```suggestion
  normalized_dnb:
    sensor: viirs
    standard_name: equalized_radiance
    operations:
      - name: linear_stretch
        method: !!python/name:satpy.enhancements.stretch
        kwargs: { stretch: 'crude', min_stretch: 0.0, max_stretch: 1.0 }
```

The section header doesn't actually mean anything beyond being a unique identifier/key. Having these two sections be the exact same thing ends up being redundant.